docs/devguide/create-module.asciidoc

```diff
@@ -77,6 +77,66 @@ the metricsets. This file does not have `_test.go` in the name because otherwise
 
 To see an example of the `testing.go` file, look at the https://github.com/elastic/beats/tree/master/metricbeat/module/mysql[mysql module].
 
+[float]
+===== Test a Metricbeat module manually
+
+To test a Metricbeat module manually, follow the steps below.
+
+First we have to build the docker image which is available for the modules. As an example let's take MySQL module.
```

Comment:
I'll add something like _The Dockerfile is located inside a `_meta` folder on each module folder_

Reply:
How about modules that don't require docker image for testing?

Reply:
We could come up with a convention that these modules have a `TESTING.md` or `README.md` inside the _meta dir on how it can be tested.